en/tween/tween-interface.md

```diff
@@ -0,0 +1,80 @@
+# Tween Interface
+
+## Properties and Interface
+
+### Interface
+
+| Interface         | Description                                     |
+| :---------------- | :------------------------------------------ |
+| **tag**           | Add a tag of type `number` to the current tween |
+| **to**            | Adds an interval action that performs **absolute value** calculations on properties |
+| **by**            | Adds an interval action that performs **relative value** calculations on properties |
+| **set**           | Add a transient action that **directly sets the target property** |
+| **delay**         | Add a **delay time** to a transient action |
+| **call**          | Add a **callback** transient
+| **target**        | Add a **directly set tween target** transient
+| **union**         | **package the contextual tween action into one** |
+| **then**          | **insert a tween into the tween queue** |
+| **repeat**        | **execute several times** (previously repeated several times, please adapt in time) |
+| **repeatForever** | **repeatForever** | **repeatForever** |
+| **sequence**      | **add a sequential execution of the tween** |
+| **parallel**      | **add a simultaneous tween** |
+| **start**         | **start tween** |
+| **stop**          | **stop tween** |
+| **clone**         | **clone tween** |
+| **show**          | **enable rendering on node chains, tween target needs to be Node** |
+| **hide**          | **disable rendering on the node chain, the tween target needs to be Node** |
+
+### Static Interface
+
+These methods are static methods of `Tween` and are called in the following example:
+
+```ts
+Tween.stopAll()
+Tween.stopAllByTag(0);
+Tween.stopAllByTarget(this.node);
+```
+
+| Interface | Description |
+| :--- | :--- |
+| **stopAll**         | Stop all tweening <br> This interface removes all registered tweens from the underlying <br> **Note**: This method affects all objects |
```

Comment:
Stop all tween motions. <br>This method will remove all registered tweens at root level. <br>**Note**: this method will affect all tween targets.